pkg/router/template/types.go

```diff
@@ -62,6 +62,7 @@ type Endpoint struct {
 	Port       string
 	TargetName string
 	PortName   string
+	IDhash     string
```

Comment:
I will make the change.
phil
On 04/04/2016 02:38 PM, Ram Ranganathan wrote:
>
> In pkg/router/template/types.go 
> <https://github.com/openshift/origin/pull/8334#discussion_r58425446>:
>
> > @@ -62,6 +62,7 @@ type Endpoint struct {
> >  	Port       string
> >  	TargetName string
> >  	PortName   string
> > +	IDhash     string
>
> |IdHash| for consistency with the other variable names. And that 
> affects the prior comments re: using |IDHash| for the server id.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openshift/origin/pull/8334/files/d5a28c792427c49fed6676cc043d4a13b621d62c#r58425446>
>